src/Core_Command.php

```diff
@@ -122,11 +122,12 @@ function check_update( $_, $assoc_args ) {
 	 */
 	public function download( $args, $assoc_args ) {
 
-		$download_dir = ! empty( $assoc_args['path'] ) ? ( rtrim( $assoc_args['path'], '/\\' ) . '/' ) : ABSPATH;
+		$download_dir      = ! empty( $assoc_args['path'] ) ? ( rtrim( $assoc_args['path'], '/\\' ) . '/' ) : ABSPATH;
 		$wordpress_present = is_readable( $download_dir . 'wp-load.php' );
 
-		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'force' ) && $wordpress_present )
+		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'force' ) && $wordpress_present ) {
```

Comment:
Not directly related to this PR, but the file already contains a `use` statement for `WP_CLI\Utils`, so this line (and all the other similar function calls) can be changed to:
```suggestion
		if ( ! Utils\get_flag_value( $assoc_args, 'force' ) && $wordpress_present ) {
```